src/app/routine-home/routine-edit/routine-edit.page.html

```diff
@@ -9,57 +9,74 @@
   </ion-toolbar>
 </ion-header>
 
-<ion-segment [scrollable]="true" [(ngModel)]="day_night_select">
-  <ion-segment-button value="day">
-    <ion-icon name="sunny-outline"></ion-icon>
-  </ion-segment-button>
-  <ion-segment-button value="night">
-    <ion-icon name="moon-outline"></ion-icon>
-  </ion-segment-button>
-</ion-segment>
+<ion-content class="ion-padding">
+  <h1 class="text-center mb-6">Routine</h1>
 
-Remind me at
-<ion-datetime presentation="time"></ion-datetime>
+  <div class="px-20 mb-6">
+    <ion-segment [scrollable]="true" [(ngModel)]="day_night_select">
+      <ion-segment-button value="day">
+        <ion-icon name="sunny-outline"></ion-icon>
+      </ion-segment-button>
+      <ion-segment-button value="night">
+        <ion-icon name="moon-outline"></ion-icon>
+      </ion-segment-button>
+    </ion-segment>
+  </div>
 
-<p>
-  Frequency
-</p>
-<ion-grid>
-  <ion-row>
-    <ion-col>S</ion-col>
-    <ion-col>M</ion-col>
-    <ion-col>T</ion-col>
-    <ion-col>W</ion-col>
-    <ion-col>Th</ion-col>
-    <ion-col>F</ion-col>
-    <ion-col>Sa</ion-col>
-  </ion-row>
-</ion-grid>
+  <div class="pt-4">
+    <h4 class="text-center mb-1">Remind me at</h4>
 
-<div *ngIf="day_night_select == 'day'">
```

Comment:
Looks like the day routine items have been removed. When I run the app the day routine does not work when adding items anymore.

Reply:
Add this code above line 51 right before the night routine
```
<div *ngIf="day_night_select == 'day'">
    <ion-item *ngFor="let routine of day_routines; index as i;">
      <ion-icon
        class="text-primary-default bigger"
        name="remove-circle-outline"
        slot="start"
        (click)="deleteRoutine(i)"
      ></ion-icon>
      <ion-input
        placeholder="Enter routine"
        [(ngModel)]="routine.label"
      ></ion-input>
    </ion-item>
  </div>
```